src/Components/FactGroup.jsx

```diff
@@ -0,0 +1,101 @@
+import { Grid, Box, Typography, makeStyles } from "@material-ui/core";
+
+const useStyles = makeStyles((theme) => ({
+  roleImage: {
+    height: 300,
+    width: 300,
+  },
+  titleRoles: {
+    padding: theme.spacing(5),
+  },
+  headingRoles: {
+    color: theme.palette.primary.light,
+    paddingBottom: theme.spacing(2),
+  },
+  roleAdjust: {
+    [theme.breakpoints.up("sm")]: {
+      paddingTop: theme.spacing(6),
+    },
+  },
+  root: {
+    marginTop: theme.spacing(5),
+    marginBottom: theme.spacing(5),
+  },
+  fact: {
+    padding: theme.spacing(1),
+  },
+}));
+
+const FactGroup = (props) => {
+  const { fact1, fact2, fact3 } = props;
```

Comment:
I've found the following pattern really helpful when rendering a dynamic number of text items:

https://github.com/openseattle/cvp/blob/staging/src/Subpages/WhyVacate/TextWithImage.jsx (lines 22-26). This way, if more facts (or less) come up down the line, we can still easily use this component.

Reply:
I thought about using that, but the problem is the asymmetric item, it only works with 3 aesthetically, so I wanted to enforce that.

Reply:
We can scrap that asymmetric item then it would be perfect. What do you think?

Reply:
Wait I think i figured out what you meant. Disregard previous.

Reply:
I think it may be more flexible if the grid item is left up to where it is called. It is lock to 2 columns right now.